plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -302,7 +302,7 @@ class GeneralServiceFormSection extends Component {
           Services
         </h2>
         <p>
-          Configure your service below. Start by giving your service a name.
+          Configure your service below. Start by giving your service an Id.
```

Comment:
I disagree - It looks strange with ID. I did try that first. @leemunroe @ashenden anyone want to weigh in here?